src/module/orbbec.cpp

```diff
@@ -49,11 +51,13 @@ const std::string kColorMimeTypeJPEG = "image/jpeg";
 const std::string kDepthSourceName = "depth";
 const std::string kDepthMimeTypeViamDep = "image/vnd.viam.dep";
 const std::string kPcdMimeType = "pointcloud/pcd";
+const std::string firmwareUrl = "https://orbbec-debian-repos-aws.s3.amazonaws.com/product/Astra2_Release_2.8.20.zip";
```

Comment:
Is expectation here that we bump this URL when we want to upgrade to a new firmware version in a new orbbec module release? No objections just yet, but just curious

Reply:
should we use a string formatter to use `requiredFirmwareVer` in this URL or is that crazy

Reply:
Could you just add a comment to update the other const for each const then? ty